tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ScalaJSClassEmitter.scala

```diff
@@ -64,6 +66,40 @@ private[scalajs] final class ScalaJSClassEmitter(
     linkedClassByName(className).kind == ClassKind.Interface
   }
 
+  private[emitter] lazy val candidateForJSConstructorOpt: Set[String] = {
+    /* Those classes appear in hard coded js files.
+     * Their constructors aren't always present in the desugared code.
+     * As such we cannot optimize those as we can't change the hard coded files.
+     */
+    val blackList = Set("jl_ClassCastException",
+      "sjsr_UndefinedBehaviorError", "jl_NullPointerException",
+      "js_CloneNotSupportedException")
```

Comment:
4 spaces.

You should be able to remove `jl_NullPointerException`. It is not in the hard-coded file anymore as of f22c743e2140498e89e50e1a158c60f7e62d6212